todo-app/src/Status.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
 
 export default class Status extends React.Component{
-    render() {
+    constructor(props) {
+        super(props);
+        this.state = {isDone: false};
+        this.click = this.click.bind(this);
+      }
+    
+      click(event) {
+        this.setState({
+          isDone: event.target.checked
+        });
+      }
+    
+      render() {
         return (
-            <div>
-                {this.props.done}
-            </div>
-        )
-    }
+          <label>
+          <input type="checkbox"
+            checked={this.state.isDone}
+            onClick={this.click}
```

Comment:
You should use `onChange` here, as it describes better what we are trying to accomplish. We want to update the `checked` we get from the state, when the actual HTML input element `checked` property changes. Does this make sense?